pom.xml

```diff
@@ -49,34 +49,27 @@
     <releaseProfiles> </releaseProfiles>
     <arguments> </arguments>
     <argLine>-Xms768M -Xmx768M -Djava.awt.headless=true -XX:+HeapDumpOnOutOfMemoryError -XX:+TieredCompilation -XX:TieredStopAtLevel=1</argLine>
-    <jenkins.version>2.138.4</jenkins.version>
-    <!-- Should only need to override these if using timestamped snapshots (but better to use Incrementals instead): -->
+
+    <jenkins.version>2.204</jenkins.version>
```

Comment:
closest thing to an LTS with the bom.  2.204 will be around in a couple of days.